wallet/i18n/de.json

```diff
@@ -76,15 +76,15 @@
   "successfully_bought_the_item": "Sie haben den Artikel erfolgreich gekauft.\\nIhr Artikel befindet sich in der Abholung",
   "product_name": "Produktname",
   "continue": "Fortsetzen",
-  "wallet_creation": "Wallet-Erstellung",
+  "wallet_creation": "Wallet Erstellung",
   "create_new_wallet": "Erstellen Sie eine neue Brieftasche",
   "proceed": "Fortfahren",
   "security_hint": "Befolgen Sie Best Practices für die Sicherheit. Achten Sie darauf, Ihre mnemonische Passphrase an einem sicheren Ort aufzuschreiben. Dieser Satz ist die einzige Möglichkeit, Ihr Konto wiederherzustellen, wenn Sie Ihr Passwort vergessen haben. Ohne Ihr Passwort oder Ihre Wiederherstellungs-Passphrase ist eine Kontowiederherstellung nicht möglich. Auch Emeris kann dir nicht helfen.",
   "import_an_account": "Importieren Sie ein Konto",
   "create_wallet": "Brieftasche erstellen",
   "terms_of_service": "Nutzungsbedingungen",
   "manage_your_nft": "VERWALTEN SIE IHR NFT",
-  "pylons_infrastructure": "Pylons ist eine schnelle NFT-Infrastruktur",
+  "pylons_infrastructure": "Pylons ist eine schnelle NFT Infrastruktur",
   "sell_and_buy_artworks": "KUNSTWERKE VERKAUFEN UND KAUFEN",
   "transactions_free": "Transaktionen sind kostenlos",
```

Comment:
Are we trying to say "Free Transactions" or "Transactions are free?". If it's transactions are free, translation is correct. Otherwise we need to fix it.